tests/cases/fourslash/completionTypeAssertion.ts

```diff
@@ -2,4 +2,4 @@
 
 //// var x = this as/*1*/
 
-verify.completions({marker: "1", exact: completion.globalsPlus(["x"]) })
+verify.completions({marker: "1", exact: completion.globals })
```

Comment:
This was a very strange test

Reply:
This test has existed before, after this modification, there should be no 'x'.

Reply:
I’m not saying there’s anything wrong here, just observing that it was weird for someone to write this test this way in the first place, a long time ago. I think the changes you made to these make sense 👍